src/sentry/integrations/slack/client.py

```diff
@@ -64,6 +66,17 @@ def authorize_request(self, prepared_request: PreparedRequest) -> PreparedReques
         prepared_request.headers["Authorization"] = f"Bearer {token}"
         return prepared_request
 
+    def is_response_error(self, resp: Response | None = None, e: Exception | None = None) -> bool:
```

Comment:
Why is `e` being passed here if it's not used?

Reply:
I meant to pass to super().is_response_error() thanks!